data/data_fixes/fix_sample_ids.rb

```diff
@@ -0,0 +1,34 @@
+require 'csv'
+require 'logger'
+
+ActiveRecord::Base.logger = nil
+logger = Logger.new("sample_ids_log.log")
+data = CSV.parse(File.read("data/data_fixes/root_sample_ids.csv"), headers: true)
+failed_ids = Array.new()
+success_ids = Array.new()
+
+data.each_with_index do |row, index|
+    if index % 1000 == 0
+        puts "."
+    end
+    samples = Sample::Metadata.where("sample_description" => row["original_root_sample_id"])
+    if samples != []
+        samples.each do |sample|
+            puts "updating: #{sample.sample_description}"
```

Comment:
I think we could also log the sampleMetadata id so we know the specific row we are changing

Reply:
why we don't write these lines in in the log file too? They seem they could be useful